POP.nix

```diff
@@ -97,7 +97,7 @@
     precedenceList = computePrecedenceList instantiator meta.supers;
     defaults = lib.foldr mergeInstance bottomInstance ([meta.defaults] ++ map getDefaults precedenceList);
     __meta__ = meta // {inherit precedenceList;};
-    proto = composeProtos ([(topProto __meta__) (extensionProto meta.extension)] ++ (map getProto precedenceList));
+    proto = composeProtos ([(extensionProto meta.extension)] ++ (map getProto precedenceList) ++ [(topProto __meta__)]);
```

Comment:
If moved at the end, isn't it a `bottomProto`, then?

Reply:
yeah makes sense, I renamed it